Package.swift

```diff
@@ -0,0 +1,26 @@
+// swift-tools-version: 5.7
+
+// NOTE! This is never released, we're only using this to support local builds builds for the
```

Comment:
I've added this package so that we can use a local dependency from the Swift SDK to this directory. This allows testing new versions of the core extension with the Swift SDK, which was was quite cumbersome before.